src/contracts/registry/registry.service.ts

```diff
@@ -118,6 +118,7 @@ export class RegistryService {
   public async getPubkeyLength(): Promise<number> {
     if (!this.cachedPubKeyLength) {
       const contract = await this.repositoryService.getCachedRegistryContract();
+      // TODO: after keys-api take from local constant
```

Comment:
I guess the whole Registry module will not be needed at all after the keys API is implemented

Reply:
I understand, but I want to delete this in keys-API branch.